chsdi/templates/htmlpopup/ivs_nat.mako

```diff
@@ -32,52 +32,57 @@
     <tr><td class="cell-left">${_('ivs_documentation')}</td>
 <%
     from urllib2 import urlopen
-    PDF_Full = c['attributes']['ivs_sortsla']
-    PDF_Level_1 =  PDF_Full[0:6] + '0000'
-    PDF_Level_1_Name = PDF_Full[0:2]+ ' ' + str(int(PDF_Full[2:6]))
-    PDF_Level_2_exist = PDF_Full[6:8]
-    PDF_Level_2 = PDF_Full[0:8] + '00'
-    PDF_Level_2_Name = PDF_Level_1_Name + '.' +  str(int(PDF_Full[6:8]))
-    PDF_Level_3_exist = PDF_Full[8:10] 
-    PDF_Level_3 = PDF_Full 
-    PDF_Level_3_Name = PDF_Level_2_Name + '.' + str(int(PDF_Full[8:10]))
-    url = "http://dav0.bgdi.admin.ch/kogis_web/downloads/ivs/beschr/de/"+c['attributes']['ivs_sortsla']+".pdf"
-    response = None
-    try:
-        response = urlopen(url)
-        pdf = True
-    except:
-        pdf = False
-    finally:
-        if response:
-            response.close()
+    from chsdi.lib.helpers import resource_exists
+    webDavHost = request.registry.settings['webdav_host']
+    pdf = None
+    if c['attributes']['ivs_sortsla'] is not None:
+        PDF_Full = c['attributes']['ivs_sortsla']
+        PDF_Level_1 =  PDF_Full[0:6] + '0000'
+        PDF_Level_1_Name = PDF_Full[0:2]+ ' ' + str(int(PDF_Full[2:6]))
+        PDF_Level_2_exist = PDF_Full[6:8]
+        PDF_Level_2 = PDF_Full[0:8] + '00'
+        PDF_Level_2_Name = PDF_Level_1_Name + '.' +  str(int(PDF_Full[6:8]))
+        PDF_Level_3_exist = PDF_Full[8:10] 
+        PDF_Level_3 = PDF_Full 
+        PDF_Level_3_Name = PDF_Level_2_Name + '.' + str(int(PDF_Full[8:10]))
+        url = webDavHost + "/kogis_web/downloads/ivs/beschr/de/" + c['attributes']['ivs_sortsla'] + ".pdf"
+        pdf = resource_exists(url)
+        response = None
+        try:
```

Comment:
you don't need this part anymore as you're using:
`pdf = resource_exists(url)` just before